style.css

```diff
@@ -79,23 +74,274 @@ p {
   height: 20px;
 }
 
-ul {
+.languages img {
+  padding-left: 20px;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+}
+
+.languages > li {
+  display: flex;
+  border: 5px solid #f7f7f9;
+  border-radius: 10px;
+  margin: 3px;
+  background: #fff;
+}
+
```

Comment:
- [OPTIONAL]  Well done with the languages 🥇 I think it would be better to use the same background color as given in the Figma design guidelines 😄 In Figma, it is `background: #F7F7F9;`


Your design now 🔽 | Figma guidelines 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/229305954-85602cb5-b581-48b6-8dde-0dfec99b609d.png) | ![image](https://user-images.githubusercontent.com/106119105/229305961-69dda8b9-7163-4586-98fa-dcc1e39c6e74.png)

Reply:
@KanzaTahreem Done.